src/util/tracking.ts

```diff
@@ -224,6 +224,7 @@ export function logEvent(event: TrackingEventName, values: TrackingValues = {}):
         const { accountReferral } = account
         params.refDeviceInstallerId = deviceReferral.installerId
         params.refDeviceCurrencyCodes = deviceReferral.currencyCodes
+        params.promoIds = accountReferral.activePromotions
```

Comment:
This would become `state.account.activePromotions`, instead of `state.account.accountReferral.activePromotions`, once you make the change requested in the previous commit.